src/chess/uciloop.cc

```diff
@@ -208,7 +203,7 @@ bool UciLoop::DispatchCommand(
   } else if (command == "quit") {
     return false;
   } else {
-    throw Exception("Unknown command: " + command);
+    return CmdNonStandardUciCommand(command, params);
```

Comment:
My idea was that `CmdNonStandardUciCommand` returns `true` if it managed to process command and `false` if not, so here it would still be:
```cpp
else if (!CmdNonStandardUciCommand(command, params)) {
  throw Exception("Unknown command: " + command);
}
```

And in `UciLoop::CmdNonStandardUciCommand` just `return false`

Reply:
Had to re-wire a few things to ensure the {"get","fen"} was moved out of the uciloop.cc file. Now indeed the typed in line will be forwarded to the engine class without command/token parsing.